docs/mcp/index.md

```diff
@@ -0,0 +1,80 @@
+# Flux MCP Server Introduction
+
+The Flux Model Context Protocol (MCP) Server connects AI assistants like Claude, Cursor,
+GitHub Copilot, and others directly to your Kubernetes clusters running [Flux Operator](../operator/index.md),
+enabling seamless interaction through natural language. It serves as a bridge between AI tools
+and your GitOps pipelines, allowing you to analyze the cluster state, troubleshoot deployment issues,
+and perform operations using conversational prompts.
+
+## Features
```

Comment:
Maybe also mention here the "feature" that the assistant could help you compose manifests for adding to your GitOps repo being aware of what's inside your cluster.

An example prompt for adding to `prompts.md`:

"Help me write a HelmRelease manifest similar to {some helmrelease} in my prod cluster with drift detection enabled and {insert here some other requirement e.g. about rollback behavior} config."